the-toad-tribune/src/layout/Politics.tsx

```diff
@@ -63,10 +77,43 @@ const PoliticsStyles = styled.div`
   background-color: grey;
   grid-area: 2/1/7/3;
   margin-left: 0.5em;
+  border-radius: 10px;
+  overflow: hidden;
 `;
 
 const Image = styled.img`
   height: 100%;
   width: 100%;
   overflow: none;
 `;
+
+const ImageContainer = styled.div`
+  max-height: 50%;
+  max-width: 100%;
+`;
+
+const PoliticsContentContainer = styled.div`
+  min-height: 50%;
+  min-width: 300px;
+  display: flex;
+  flex-direction: column;
+  margin: .4rem .6rem .6rem .6rem;
+`;
+
+const PoliticsTitle = styled.h3`
+  margin: .3rem;
+`;
+
+const PoliticsAuthors = styled.h4`
+  margin: .4rem;
+  
+`;
+
+const PoliticsPublishedDate = styled.h5`
+margin: 0 .4rem .2rem .4rem;
+`;
+
+const PoliticsDescription = styled.p`
+`;
```

Comment:
You can also nest the H and P tags inside PoliticsContentContainer like how you would with SCSS. 

Example:
``` 
const PoliticsContentContainer = styled.div`
  min-height: 50%;
  min-width: 300px;
  display: flex;
  flex-direction: column;
  margin: .4rem .6rem .6rem .6rem;

         h3 {
                margin.3rem;
              }
`;

```

Reply:
Thanks tol!

Reply:
Nice. Good to know. That will tidy things up quite a bit.